draft-terminology.md

```diff
@@ -180,8 +180,8 @@ informative:
 
 --- abstract
 
-This document argues for moving away from specific language conventions used by RFC authors and RFC Editors in order to encourage inclusive terminology in the ongoing RFC 
-series. The document also provides examples of inclusive terminology as precise alternatives for these conventions. 
+This document argues for encouraging inclusive terminology in the ongoing RFC 
```

Comment:
Tried to make this sentence clearer - "This document argues for moving away from specific language conventions used by RFC authors and RFC Editors in order to encourage inclusive terminology in the ongoing RFC 
series" sounded like the language conventions were being used to encourage inclusive terminology (on first read)